packages/story-editor/src/components/canvas/framesLayer.js

```diff
@@ -41,12 +44,30 @@ import { Layer, NavNextArea, NavPrevArea, PageArea } from './layout';
 import FrameElement from './frameElement';
 import Selection from './selection';
 import PageNav from './pagenav';
+import {
+  FOCUS_GROUPS,
+  useEditLayerFocusManager,
+} from './editLayerFocusManager';
 
 const FramesPageArea = styled(PageArea)`
   pointer-events: initial;
 `;
 const marginRatio = 100 * (DESIGN_SPACE_MARGIN / PAGE_WIDTH);
 
+const FocusContainer = styled.div`
+  grid-row: 2 / -1;
+  grid-column: 1 / -1;
```

Comment:
nit: a comment here about the header could be helpful for our future selves, but totally not necessary